deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -101,66 +76,35 @@ def compact_partition(params: CompactPartitionParams, **kwargs) -> Optional[str]
             params,
             **kwargs,
         )
-        compaction_session_type: str = (
-            "INPLACE"
-            if execute_compaction_result.is_inplace_compacted
-            else "NON-INPLACE"
-        )
-        logger.info(
-            f"Partition-{params.source_partition_locator} -> "
-            f"{compaction_session_type} Compaction session data processing completed"
-        )
-        if execute_compaction_result.new_compacted_partition:
-            previous_partition: Optional[Partition] = None
-            if execute_compaction_result.is_inplace_compacted:
-                previous_partition: Optional[
-                    Partition
-                ] = params.deltacat_storage.get_partition(
-                    params.source_partition_locator.stream_locator,
-                    params.source_partition_locator.partition_values,
-                    **params.deltacat_storage_kwargs,
-                )
-                # NOTE: Retrieving the previous partition again as the partition_id may have changed by the time commit_partition is called.
-            logger.info(
-                f"Committing compacted partition to: {execute_compaction_result.new_compacted_partition.locator} "
-                f"using previous partition: {previous_partition.locator if previous_partition else None}"
-            )
-            committed_partition: Partition = params.deltacat_storage.commit_partition(
-                execute_compaction_result.new_compacted_partition,
-                previous_partition,
-                **params.deltacat_storage_kwargs,
-            )
-            logger.info(f"Committed compacted partition: {committed_partition}")
-        else:
-            logger.warning("No new partition was committed during compaction.")
-
-        logger.info(
-            f"Completed compaction session for: {params.source_partition_locator}"
-        )
-        return execute_compaction_result.round_completion_file_s3_url
+        return _upload_compaction_result(params, execute_compaction_result)
 
 
 def _execute_compaction(
     params: CompactPartitionParams, **kwargs
 ) -> ExecutionCompactionResult:
-
-    rcf_source_partition_locator = (
+    compaction_start_time: float = time.monotonic()
```

Comment:
Definitely add one test to check if all audit info is being set.

Reply:
Any update?

Reply:
Yes, that's correct. You can check what properties are set by skimming through the code.